CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 
+* Move from Braintree to PayPal analytics service (FPTI)
```

Comment:
Should this have a public CHANGELOG line? Seems internal maybe - except is it now required for all merchants to send analytics where it previously was optional?

Reply:
Mostly added it for our own tracking of when the switch was made for easier tracking (also to align w/ iOS)